### ISSUE TITLE
Latest version has regex issue

### ISSUE THREAD

**Issue by chaimpeck** [johngrogg/ics-parser#26]
My ical file has two lines that look like this:
```
DTSTART;TZID=America/New_York:20150601T064500
DTEND;TZID=America/New_York:20150601T073000
```

The output looks like this:

```
Array
(
    [York] => 20150601T073000
    [York_array] => Array
        (
            [0] => 20150601T064500
            [1] => 20150601T073000
        )
    ...

```

I was able to resolve this by modifying the regex here:

```
public function keyValueFromString($text)
    {
        preg_match('/([A-Za-z-\/;=^:]+)[:]([\w\W]*)/', $text, $matches);
        if (count($matches) == 0) {
            return false;
        }
        $matches = array_splice($matches, 1, 2);
        return $matches;
    }
```

If I change that to this, it works:

```
preg_match('/([^:]+)[:]([\w\W]*)/', $text, $matches);
```

Is there any reason why it is not that?

---

I hope this helps. And this is an awesome project. Thank you for making an maintaining it.

**Comment by u01jmg3** [comment]
Investigating this issue and looking at other issues surrounding the regex that's in use

**Comment by u01jmg3** [comment]
Closed with 4922899

**Comment by u01jmg3** [comment]
Please let me know of any issues
